speculation-rules.bs

```diff
@@ -586,7 +604,7 @@ A <dfn>prerender candidate</dfn> is a [=struct=] with the following [=struct/ite
   1. End the [=synchronous section=], continuing the remaining steps [=in parallel=].
   1. [=list/For each=] |prefetchCandidate| of |prefetchCandidates|:
     1. The user agent may run the following steps:
```

Comment:
The existing spec text seems general enough to cover the incorporation of NVS hint into user agent heuristics (for prefetching a non-eager candidate when a predicted URL inexactly matches the candidate). But perhaps a non-normative note near the eagerness description on line 616 would be useful?

Reply:
+1 to a non-normative note.

Reply:
Done.